app/components/permissions_list.rb

```diff
@@ -2,4 +2,13 @@ class PermissionsList < ViewComponent::Base
   def initialize(permission_model)
     @permission_model = permission_model
   end
+
+  def view_applications_only?
```

Comment:
I like this but the list of user-level permissions may change in the future and this method could fall behind. How about we move it to `app/models/provider_permissions.rb` and use `VALID_PERMISSIONS` like so:

```ruby
  def view_applications_only?
    VALID_PERMISSIONS.map { |permission| send(permission) }.all?(false)
  end
```